Tools/clinic/clinic.py

```diff
@@ -3336,9 +3337,8 @@ def parser_name(self) -> str:
 
     def add_include(self, name: str, reason: str,
                     *, condition: str | None = None) -> None:
-        if self.include is not None:
-            raise ValueError("a converter only supports a single include")
-        self.include = Include(name, reason, condition)
+        include = Include(name, reason, condition)
```

Comment:
We should remove duplicates here, when the filename is the same. If you want to allow multiple includes per converter, move Clinic.add_include() logic in a new Includes class which is a list but avoids duplicates. It may handle merging includes from converter for Clinic.

Reply:
I defer that to `Clinic.add_include`; this is sinply a list of includes. When the final includes are determined in Clinic, duplicates are handled.